dbt/parsely_dpl/historical_script.py

```diff
@@ -0,0 +1,61 @@
+
+
+import datetime
+import argparse
+import subprocess
+import os
+import parsely_raw_data.redshift as redshift
+
+def daterange(d1, d2):
+    return (d1 + datetime.timedelta(days=i) for i in range((d2 - d1).days + 1))
```

Comment:
This will be specific to the time of the computer/server that the script is being run on - I don't think this is something we can control. It will match the date inputted with the S3 bucket date. Which may be in UTC.